smartsim/_core/launcher/colocated.py

```diff
@@ -46,7 +39,7 @@ def write_colocated_launch_script(
     :param colocated_settings: fs settings from entity run_settings
     """
 
-    colocated_cmd = _build_colocated_wrapper_cmd(fs_log, **colocated_settings)
+    colocated_cmd = ""
 
     with open(file_name, "w", encoding="utf-8") as script_file:
```

Comment:
@MattToast If we were to not support colocated deployments of applications for HPC WLMs, can we also remove this file if we remove the entrypoint?

Reply:
It looks like it! The only thing that we actually export out of this namespace is just a function to build a colocate script, but since we're not planning on standing up any MLI this way, this module should be free to be removed.